lib/PhpZendClientGenerator.php

```diff
@@ -431,13 +431,21 @@ function writeClass(DOMElement $classNode)
 					break;
 					
 				case "string" :
-					$this->appendLine("		if(count(\$xml->{$propName}))");
-					$this->appendLine("		{");
-					$this->appendLine("			if(isset(\$xml->{$propName}->item) && count(\$xml->{$propName}->item))");
-					$this->appendLine("				\$this->multiLingual_{$propName} = Kaltura_Client_ParseUtils::unmarshalArray(\$xml->$propName, '');");
-					$this->appendLine("			else");
-					$this->appendLine("				\$this->$propName = ($propType)\$xml->$propName;");
-					$this->appendLine("		}");
+					if($propertyNode->getAttribute("multiLingual") == 1)
```

Comment:
add this condition like the isEnum on line 407
then check for if(isMultiLingual)